app/models/metadata_revision/featured_attachment_ordering_validator.rb

```diff
@@ -0,0 +1,12 @@
+class MetadataRevision::FeaturedAttachmentOrderingValidator < ActiveModel::Validator
+  ATTRIBUTE = :featured_attachment_ordering
+  ORDER_ITEM_REGEX = /^FileAttachment\d+$/.freeze
```

Comment:
Sure, I've updated with `\A` and `\Z`. My assumption was `$` and `^` are more familiar, but the former pair are more correct.

I'm going to leave the ID format as-is for now, since we're not expecting to be reading it often. I like that it's the shortest thing that works.